ckanext/usmetadata/plugin.py

```diff
@@ -154,6 +154,7 @@
         r'^([Dd]ecennial)|([Qq]uadrennial)|([Aa]nnual)|([Bb]imonthly)|([Ss]emiweekly)|([Dd]aily)|([Bb]iweekly)'
         r'|([Ss]emiannual)|([Bb]iennial)|([Tt]riennial)|([Tt]hree times a week)|([Tt]hree times a month)'
         r'|(Continuously updated)|([Mm]onthly)|([Qq]uarterly)|([Ss]emimonthly)|([Tt]hree times a year)'
+        r'|^R\/P(?:(\d+(?:[\.,]\d+)?)Y)?(?:(\d+(?:[\.,]\d+)?)M)?(?:(\d+(?:[\.,]\d+)?)D)?(?:T(?:(\d+(?:[\.,]\d+)?)H)?(?:(\d+(?:[\.,]\d+)?)M)?(?:(\d+(?:[\.,]\d+)?)S)?)?$' # ISO 8601 duration
```

Comment:
Can you remove the others? According to the [spec](https://resources.data.gov/resources/dcat-us/#accrualPeriodicity), only `irregular` or an ISO 8601 duration are acceptable for this field.

Reply:
@adborden we intentionally left it so, because we don't touch the old form.  We apply these changes on the new form, but the old form stays the same, which use values like Decennial, Annual etc. 

My initial intention was also to remove those old validations, but then thought we might need to keep the old form working.

Reply:
Okay, let's open an issue or leave a TODO comment to remove them once the old form is removed.